### DISCUSSION TITLE
توضیح نسخه‌های متفاوت

### DISCUSSION THREAD

**Discussion by RtinLFS** [MDarvishi5124/Arad#4] [Q&A] [answered]
درود و عرض ادب.
اگر امکانش هست در خصوص نسخه‌های متفاوت در README پروژه توضیح دهید. اینکه نسخه‌های webfont, variable و static چه تفاوت‌هایی دارند. اینکه چه فرقی هست بین non-english و farsi-digit
به نظرم کمک خیلی کاملی می‌کنه به مخاطبین برای انتخاب نسخه درست فونت. 

سپاس از شما

**Comment by MDarvishi5124** [comment]
سلام، کمی سرم شلوغه. قصد دارم ان‌شاءالله برای انتشار نسخۀ اصلی یه راهنما جامع‌تر بنویسم. ولی این رو همین الان به readme پروژه اضافه می‌کنم.

**Comment by MDarvishi5124** [comment, marked as answer]
تو سایت جدید، این موضوع، و سوالات دیگه، تو بخش راهنما قرار گرفته.
